introduction.md

```diff
@@ -1,3 +1,10 @@
 This is me
 
 My favorite Code School path is Javascript
+
+============
+
+My Deadly skills
+* Git
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?